### ISSUE TITLE
Error: Invalid digit found in string

### ISSUE THREAD

**Issue by archiif** [kaegi/alass#28]
This particular subtitle file caused a "Invalid digit found in string" error, is there any way to fix it?
```
error: processing subtitle file 'D:\[Asakura]Yuru_Camp_-_08[1080p][F77BB549].srt' failed
caused by: parsing subtitle file 'D:\[Asakura]Yuru_Camp_-_08[1080p][F77BB549].srt' failed
caused by: parsing the subtitle data failed
caused by: parse error at line `5086`
caused by: expected SubRip index line, found '<font size="15">- Do not soak in water.'
caused by: invalid digit found in string
```
I've included the subtitle file in the attachments:
[[Asakura]Yuru_Camp_-_08[1080p][F77BB549].zip](https://github.com/kaegi/alass/files/5899432/Asakura.Yuru_Camp_-_08.1080p.F77BB549.zip)

**Comment by Granshmeyr** [comment]
I wanted to respond to this for reference because I had the same issue. Basically this error is caused by a broken subtitle file which therefore cannot be parsed by alass.

I'm not sure why but when I used SubtitleEdit to convert the embedded .ass subtitle track in anime.mkv to an output.srt file, it broke the subtitles. One solution was to convert the embedded .ass subtitle track to output.ass which as expected did not break the syntax of the subtitle file. This is fine because alass supports .ass files. The other solution was to "Unbreak" the broken lines in the output.srt file using SubtitleEdit which seemed to work fine but will probably make the subtitle styling look off.

TL;DR your subtitle file has broken syntax.
